assets/templates/sidebar.hbs

```diff
@@ -0,0 +1,5 @@
+
+    {{#each this as |post|}}
+        <a href="{{# id}}">{{ post.title }}</a>
```

Comment:
Move the # outside of the {{ or you will have a compile error and the template will be undefined